Source/relay/TourGuide/Items of the Month/2023/Closed Circuit Pay Phone.ash

```diff
@@ -0,0 +1,87 @@
+RegisterTaskGenerationFunction("IOTMClosedCircuitPayPhoneGenerateTasks");
+void IOTMClosedCircuitPayPhoneGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries)
+{
+    if (!lookupItem("closed-circuit pay phone").have())
+        return;
+
+    string [int] affinityDescription;
+    string [int] questDescription;
+    string url = "inv_use.php?pwd=" + my_hash() + "&which=3&whichitem=11169";
+
+    int shadowLodestones = available_amount($item[Rufus's shadow lodestone]);
+    if (shadowLodestones > 0) {
+        questDescription.listAppend(HTMLGenerateSpanFont("Have " + pluralise($item[Rufus's shadow lodestone]) + ".", "purple"));
+    }
+
+    int riftAdvsUntilNC = get_property_int("encountersUntilSRChoice");
+    questDescription.listAppend(HTMLGenerateSpanFont(riftAdvsUntilNC + " encounters until NC/boss.", "black"));
+
+    if (get_property("questRufus").contains_text("step1")) {
+        questDescription.listAppend(HTMLGenerateSpanFont("Call Rufus and get a lodestone", "black"));
+        task_entries.listAppend(ChecklistEntryMake("__item closed-circuit pay phone", url, ChecklistSubentryMake("Rufus quest done", "", questDescription), -11));
+    }
+    else if (get_property("questRufus").contains_text("unstarted")) {
+        questDescription.listAppend(HTMLGenerateSpanFont("Have no quest accepted.", "red"));
+        optional_task_entries.listAppend(ChecklistEntryMake("__item closed-circuit pay phone", url, ChecklistSubentryMake("Rufus quest doable now", "", questDescription), 11));
+    }
+    else if (get_property("questRufus").contains_text("started")) {
+        optional_task_entries.listAppend(ChecklistEntryMake("__item closed-circuit pay phone", url, ChecklistSubentryMake("Rufus quest in progress", "", questDescription), 11));
+    }
+
+    if (riftAdvsUntilNC == 0) {
+        questDescription.listAppend(HTMLGenerateSpanFont("Fight a boss or get an artifact", "black"));
+        task_entries.listAppend(ChecklistEntryMake("__item shadow bucket", url, ChecklistSubentryMake("Shadow Rift NC up next", "", questDescription), -11));
+    }
+
+    if ($effect[Shadow Affinity].have_effect() > 0) {
+        int shadowRiftFightsDoableRightNow = $effect[Shadow Affinity].have_effect();
+        affinityDescription.listAppend(HTMLGenerateSpanFont("Shadow Rift fights are free!", "purple"));
+        affinityDescription.listAppend(HTMLGenerateSpanFont("(don't use other free kills in there)", "black"));
```

Comment:
should we change this to "don't use other free kills in there" and "don't spend non-free turns outside of rifts"?

Reply:
I think the tile itself is your reminder to not spend non-rift turns. The helper text can be removed to save space.

Reply:
I left this as-is for now since there wasn't too much consensus here but happy to tweak further if we want.

Reply:
that's fine